server/go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/kennygrant/sanitize v1.2.4
 	github.com/labstack/echo/v4 v4.11.4
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/oklog/ulid v1.3.1
```

Comment:
why is this direct dependency now?

Reply:
Okay, I'll put it back.
For some reason, in my environment, it happens when I run `make gql`.